elementary/monitor/dbt_project/macros/get_test_rows_sample.sql

```diff
@@ -1,18 +1,13 @@
-{%- macro get_test_rows_sample(test, test_results_query, test_type, results_sample_limit = 5) -%}
-    {% set result_rows = test.result_rows %}
-    {% if result_rows is defined and result_rows is not none %}
-      {% do return(fromjson(result_rows)) %}
-    {% endif %}
-
-    {% set test_rows_sample = none %}
-    {% if test_results_query %}
-        {% set test_rows_sample_query = test_results_query %}
+{%- macro get_test_rows_sample(result_rows_agate, test_type, results_sample_limit=5) -%}
```

Comment:
I assume we're going for a minor bump, right? (as this code is not backwards compatible)

Reply:
Are you referring to our usage of `test_result_rows` model?

Reply:
Yeah
Let's discuss quickly on slack